.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,38 @@
+name: Benchmarks
+
+on:
+  workflow_call:
+    inputs:
+        linux_5_8_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux 5.8 Swift version matrix job. Defaults to true."
+            default: true
+        linux_5_9_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux 5.9 Swift version matrix job. Defaults to true."
+            default: true
+        linux_5_10_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux 5.10 Swift version matrix job. Defaults to true."
+            default: true
+        linux_nightly_6_0_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux nightly 6.0 Swift version matrix job. Defaults to true."
+            default: true
+        linux_nightly_main_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux nightly main Swift version matrix job. Defaults to true."
+            default: true
+ 
+jobs:
+    benchmarks:
+        name: Benchmarks
+        uses: ./.github/workflows/pull_request_swift_matrix.yml
+        with:
+          name: "Benchmarks"
+          matrix_linux_command: "apt-get update -y -q && apt-get install -y -q libjemalloc-dev && swift package --package-path Benchmarks/ --disable-sandbox benchmark baseline check --check-absolute-path Benchmarks/Thresholds/${SWIFT_VERSION}/"
```

Comment:
Should we make the thresholds root directory an input with default?

Reply:
Is it OK to have `--disable-sandbox` be default?

Reply:
Removed the `--disable-sandbox` we don't need it on the CI.